### PR TITLE
cleanup: adding DRIFT region tags to yaml and sh files

### PR DESCRIPTION
**Pull request by ckim328**
### Fixes 
n/a

#### Description
Ongoing effort of adding region tags to our yaml and shell files in our repos. Allows for our team to leverage internal tool

#### Change summary
* Added region tags. Snippet-bot will add a comment below of the tags added (and where).
* Also updated the tf files that had `anthos_bm` to be wrapped with `anthosbaremetal` tags. Will create a CL to update our docs. 

#### Related PRs/Issues
Similar PRs can be found here [microservice-demo PR](https://github.com/GoogleCloudPlatform/microservices-demo/pull/684), [bank-of-anthos PR](https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/611)

### CODE REVIEW

File: attached-logging-monitoring/monitoring/prometheus.yaml [hunk 1]
```diff
@@ -35,7 +38,9 @@ rules:
   - get
   - list
   - watch
+# [END anthos_monitoring_prometheus_clusterrole_stackdriver_user:prometheus]
```
Comment (askmeegs) [inline attached-logging-monitoring/monitoring/prometheus.yaml:41]: are we allowed to have semicolons (`:`) in region tags?

**Review by askmeegs — CHANGES_REQUESTED** [review]
Looks great! One quick thing, may be worth replacing `:` with `_` in any tag with a semicolon.

File: attached-logging-monitoring/monitoring/prometheus.yaml (hunk 1 above)
Comment (ckim328) [reply attached-logging-monitoring/monitoring/prometheus.yaml]: whoops!! good catch! 🦅

File: test/integration/abm_gce_defaults_on_owner_project/inspec.yml [hunk 2]
```diff
@@ -12,23 +12,25 @@
...
-  - name: vxlan_check
-    required: true
-    type: string
-  - name: ssh_as_tfadmin
-    required: true
-    type: string
-  - name: abm_install_check
-    required: true
-    type: string
+- name: project_id
```
Comment (Shabirmean) [inline test/integration/abm_gce_defaults_on_owner_project/inspec.yml:18]: Let's make sure the formatting change doesn't impact the tests.

**Review by Shabirmean — APPROVED** [review]
LGTM!

File: test/integration/abm_gce_defaults_on_owner_project/inspec.yml (hunk 2 above)
Comment (ckim328) [reply test/integration/abm_gce_defaults_on_owner_project/inspec.yml]: I just reverted this! Id rather not mess with it, thanks for catching that! 👀

**Review by askmeegs — APPROVED** [review]

**Pull request merged**